bulbs/special_coverage/views.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django.http import Http404
+from django.views.generic import DetailView
+from django.shortcuts import get_object_or_404
+
+from bulbs.special_coverage.models import SpecialCoverage
+from bulbs.content.views import BaseContentDetailView
+
+
+class SpecialCoverageView(BaseContentDetailView):
+    redirect_correct_path = False
+
+    def show_published_only(self):
+        return bool("full_preview" not in self.request.GET)
+
+    def get_template_names(self):
+        template_names = ["special_coverage/default.html"]
+
+        extended = getattr(settings, 'SPECIAL_COVERAGE_LANDING_TEMPLATE', None)
```

Comment:
ehhhhh. I don't like the pattern of having every template being configurable. We used it for the custom template logic since that varied a bit and is non-standard by nature, but this should be standard or the view should be overwritten